Dockerfile

```diff
@@ -12,24 +12,16 @@ ENV KBOT 1.9.1
 
 
 ### 2. Get Java and all required system libraries
-RUN apt-get update && apt-get upgrade -y \
- && apt-get install -y software-properties-common \
-  build-essential git \
-  openjdk-8-jre openjdk-8-jdk
+RUN apt-get update && apt-get upgrade -y
+RUN apt-get install -y software-properties-common build-essential openjdk-8-jre openjdk-8-jdk git make curl wget tar
```

Comment:
It _is_ better to put both `apt-get` commands into the same chain connected by `&&`. I.e., if `apt-get update` fails, there is no reason to attempt any `install` commands.

Reply:
Also, I recommend to put each of the packages on a line (not a `RUN`) by itself. Makes it way easier to read, and version control can easily show what is later added or removed.